bindings/protocols/coap/index.html

```diff
@@ -54,86 +54,279 @@ <h2>Introduction</h2>
+            "forms": [{
+                "href": "coap://[2001:DB8::1]/status",
+                "contentType": "text/plain;charset=utf-8"
+            }]
+        }
+    }
+}
+</pre>
+            <p>
+                The <code>contentType</code> and <code>contentCoding</code> members of a form specify the CoAP content format of the request and/or response.
+                Consumers are expected to be able to map the combinations of content type and content coding they understand to the corresponding CoAP content format IDs listed in the <a href="https://www.iana.org/assignments/core-parameters/#content-formats">IANA CoAP Content-Formats registry</a>.
+            </p>
+        </section>
+        <section>
+            <h3>Content Negotiation</h3>
+            <p>
+                Content negotiation in CoAP is used to negotiate the representation of CoAP resources that may have different representations available.
+            </p>
+            <p>
+                Content negotiation is accomplished through the use of CoAP <code>Accept</code> and <code>Content-Format</code> options.
+                The CoAP Accept option is used by clients to request a particular content format, while the <code>Content-Format</code> option is used by clients and servers to indicate the content format of the representation in requests and responses, respectively.
+            </p>
+            <p>
+                The ... can be used to provide a list of one or more content formats that the Thing is able to provide in a CoAP response for the operation.
+                The Consumer selects the most suitable one and sets the <code>Accept</code> option in the request accordingly.
+            </p>
+            <pre id="example-request-accept" class="example" title="...">
+{
+    ...
+}
+</pre>
+            <p>
+                The ... can be used to provide a list of one or more content formats that the Thing is willing to accept in a CoAP request for the operation.
+                The Consumer selects the most suitable one and sets the <code>Content-Format</code> option in the request accordingly.
+            </p>
+            <pre id="example-request-format" class="example" title="...">
+{
+    ...
+}
+</pre>
+        </section>
+        <section>
+            <h3>Caching</h3>
+            <p>
+                CoAP features two mechanisms to make efficient use of network resources and minimize the number of required transmissions.
+                First, a CoAP client may store responses to requests in order to satisfy future requests without sending a new request.
+                A stored response may be used if it is still valid, i.e., if the response's Max-Age value has not been exceeded.
+                Second, a CoAP client may validate a stored response by sending a new request with the ETag from the stored response.
+                This allows the server to quickly respond with a 4.03 (Valid) response if the stored response is still valid.
```

Comment:
I think the code should be 2.03 here (c.f. https://datatracker.ietf.org/doc/html/rfc7252#section-5.9.1.3).

```suggestion
                This allows the server to quickly respond with a 2.03 (Valid) response if the stored response is still valid.
```